quantities/measures/Build.cs

```diff
@@ -12,7 +12,7 @@ internal static class Build<TMeasure> where TMeasure : IMeasure
     public static Quant With<TInjector>(in Double value)
         where TInjector : IInjector, new()
     {
-        return new(value, in MapPool<TInjector>.Item);
+        return new(in value, in MapPool<TInjector>.Item);
```

Comment:
You mean instead of `With`?

I think I started out like that when the class was still called "Builder". Usage then looked like this:

```csharp
Builder<TMeasure>.Create<TInjector>(42);
```
which IMHO looks super weird with those type parameters.

Compare this to:

```csharp
Build<TMeasure>.With<TInjector>(42);
``` 
Much closer to an English sentence. What do you think?

IMHO the only remaining weirdness is the value we pass at the very end....

Reply:
I was wondering why you didn't use `Create` instead of `new`. I didn't understand at the time when you use the factory to create a new instance and when you use a simple instantiation. Now that I think of it, it's still a bit fuzzy 😅